### Eclipse Workspace Patch 1.0
#P apydia
Index: apydia/command.py
===================================================================
--- .	(revision 45)
+++ .	(working copy)
@@ -47,7 +47,9 @@
         open = Boolean("o", "open generated files in browser when done"),
         config = Filename("c", "specify config file"),
         verbose = Boolean("v", "be verbose"),
-        apiprefix = String("p", "namespace prefix for generated api files")
+        apiprefix = String("p", "namespace prefix for generated api files"),
+        exclude_gtk = Boolean("g", "exclude GTK/GObject methods",
+                              default=False)
     )
     
     def parse_config_file(self, options, args):
Index: apydia/descriptors.py
===================================================================
--- .	(revision 45)
+++ .	(working copy)
@@ -337,6 +337,14 @@
         for key in keys:
             try:
                 value = getattr(self.value, key)
+                if self.project.options.exclude_gtk:
+                    if hasattr(value, "__gtype__") \
+                    or (hasattr(value, "__objclass__") \
+                        and hasattr(value.__objclass__, "__gtype__")) \
+                    or (hasattr(value, "__class__") \
+                        and value.__class__.__name__ == "GProps") \
+                    or "GObjectMeta" in repr(value):
+                        continue
                 desc = create_desc(value, self.project)
                 desc.parent = self
                 results[key] = desc
